### DISCUSSION TITLE
Weired behaviour of SMARTS ring opening

### DISCUSSION THREAD

**Discussion by GurevichPE** [rdkit/rdkit#7769] [Q&A]
Hi again :) 
I have a Ni with bidentat ligand. I want to encode SMARTS pattern to break just one Ni - ligand bond. 
I did and got this:
![image](https://github.com/user-attachments/assets/1d6f8b95-7c1b-48fb-aa6c-dcadbdf1d657)

But I want this:
![image](https://github.com/user-attachments/assets/e98f15a1-8cb9-4fe4-8b31-8215dcdca21f)

Is there a way to encode reaction correctly but without writing all atoms in the chain?

Also I should highlight, that this pattern works chemically incorrect both for rings with and without dative bonds. 
![image](https://github.com/user-attachments/assets/863567f2-1b43-4b44-837a-593afdac7887)

Thank you so much!

**Comment by greglandrum** [comment]
The solution to this is discussed in multiple places that a web search would have turned up.
Here's one of them: https://github.com/rdkit/rdkit/issues/5159
